lib/src/model.dart

```diff
@@ -507,7 +507,7 @@ class Library extends ModelElement {
 
   String get path => _library.definingCompilationUnit.name;
 
-  String get nameForFile => name.replaceAll(':', '-');
```

Comment:
If we change this to fileName, it should return the full filename... `libname-library.html`. Could we do that?